### DISCUSSION TITLE
Representação da APyB perante fornecedor

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#127] [Geral]
Olá, pessoal! Tudo bem?

Estamos num impasse em relação ao hotel que estamos reservando para Keynotes/etc da Python Cerrado: o hotel está exigindo uma confirmação da representante atual da APyB (@anicelysantos) para aceitar nosso pedido de geração de nota fiscal.
Já tentamos negociar essa exigência, mas eles estão realmente irredutíveis, por isso gostaríamos de saber para qual e-mail encaminhar essa solicitação de reserva e geração da nota fiscal.
Posso usar o presidencia@python.org.br? Ou o financeiro@python.org.br? Preferem que seja enviado para o contato@python.org.br?
O hotel se encarregará de encaminhar um e-mail para o contato que fornecermos para fechar essa negociação.

**Comment by anicelysantos** [comment]
Oi @anadulce 
Pode enviar no diretoria@python.org.br

**Comment by cecivieira** [comment]
@anadulce a reserva que o hotel quer a gente assine é para 30 pessoas (R$98.735,00). Esse é o contrato?
Qual é o e-mail da python cerrado? Assim, colocarei vocês em thread.

Por experiências anteriores, a organização faz todo o intermédio com a fornecedora e nos envia o contrato pra assinatura, quando precisa. Nesse caso o hotel está falando diretamente conosco, que não temos contexto e nem sabemos se ele corresponde aos termos alinhados com a organização da Python Cerrado.

Sugiro que vocês façam todos os alinhamentos necessários, leitura do contrato e nos envie para assinatura.

**Reply by rafahela** [reply]
Boa tarde, o contrato foi analisado e está apto para ser assinado.

**Comment by cecivieira** [comment]
@rafahela o contrato é no valor de R$98.735,00
O evento, até agora, só tem R$550,00 a receber. 
A APyB não tem em caixa tal valor para ser destinado à Python Cerrado. 
É necessário que vocês tenham patrocínio que cubra o valor desse contrato, ou seja, após o recebimento de valor aproximado que cubra esse custo, a APyB assinará o contrato.

**Reply by ericof** [reply]
Olá @cecivieira,

Primeiramente peço desculpas pelos desencontros ocorridos até o momento. Eu estava a frente da negociação com o hotel, mas como ela não se resolveu antes das minha férias parte das informações acabaram se perdendo.

Queria informar que já temos o valor de patrocínio necessário para cobrir esses custos. Até o momento emitimos duas notas para empresas brasileiras (conforme tarefas abertas aqui no GitHub), mas Os pagamentos de patrocínio feitos com empresas de fora do Brasil já foramfaturados via Plone Foundation, e estão disponíveis em USD.  Posso te enviar os comprovantes se necessário for.

Esse contrato é importante para nós pois está bloqueando a obtenção de visto de 4 participantes da conferência (1 egipcio e 3 indianos).

A última versão do contrato contém um erro que estamos esclarecendo, mas de resto finalmente alinhamos o que era necessário.

Grato,
EA

**Reply by cecivieira** [reply]
@ericof devido ao alto valor do contrato (em trono de R$99mil)  frente às notas fiscais já emitidas de patrocínio (totalizando em torno de R$5mil) , esse dinheiro de patrocínio da Plone precisaria estar disponível no caixa da APyB para que possamos assinar o contrato.

**Reply by cecivieira** [reply]
@ericof os dados bancários da APyB para recebimento de moeda estrangeira, estão disponíveis em: https://apyb.python.org.br/associados/guias/como-organizar-eventos/#quais-sao-os-dados-da-pessoa-juridica-da-apyb

Neste mesmo Guia também há o link para emissão de Invoice, documento contábil necessário para registrar essa movimentação financeira.

**Reply by ericof** [reply]
@cecivieira O valor foi reduzido para 52k.

**Reply by cecivieira** [reply]
@ericof Obrigada pela informação.
Seguimos com a mesma orientação anterior.

**Reply by ericof** [reply]
@cecivieira, Dada a urgência dessa questão, vamos buscar outra alternativa então.
